src/main/scala/dotty/dokka/tasty/ScalaDocSupport.scala

```diff
@@ -48,13 +49,39 @@ trait ScaladocSupport { self: TastyParser =>
     inline def addOpt(opt: Option[dkkd.DocTag])(wrap: dkkd.DocTag => dkkd.TagWrapper) =
       opt.foreach { t => bld.add(wrap(t)) }
 
-    inline def addSeq(seq: Seq[dkkd.DocTag])(wrap: dkkd.DocTag => dkkd.TagWrapper) =
+    inline def addSeq[T](seq: Iterable[T])(wrap: T => dkkd.TagWrapper) =
       seq.foreach { t => bld.add(wrap(t)) }
 
+    // this is a total kludge, this should be done in a deeper layer but we'd
+    // need to refactor code there first
+    def correctParagraphTags(tag: dkkd.DocTag): dkkd.DocTag =
+      tag match {
+        case tag: dkkd.P =>
+          // NOTE we recurse once, since both the top-level element and its children can be P
+          // (there is no special root DocTag)
+          dkkd.Span(tag.getChildren.iterator.asScala.map(correctParagraphTags).toSeq.asJava, tag.getParams)
+        case tag => tag
+      }
+
     addSeq(parsed.authors)(dkkd.Author(_))
     addOpt(parsed.version)(dkkd.Version(_))
     addOpt(parsed.since)(dkkd.Since(_))
+    addOpt(parsed.deprecated)(dkkd.Deprecated(_))
+    addSeq(parsed.todo)(dkkd.CustomTagWrapper(_, "Todo"))
+    addSeq(parsed.see)(dkkd.CustomTagWrapper(_, "See"))
```

Comment:
In dokka there is special TagWrapper for `See`

Reply:
Right, that's a good catch. I'll actually need to do more work here - all of these tags should be repeatable, but currently all NamedTags with the same name are deduplicated. This is also why I didn't use the original Dokka classes - they are all named, but deduplicating them is not at all what we want.